packages/api/tests/es/test-stats.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const test = require('ava');
+const rewire = require('rewire');
+
+const awsServices = require('@cumulus/aws-client/services');
+const s3 = require('@cumulus/aws-client/S3');
+const { randomId } = require('@cumulus/common/test-utils');
+
+const indexer = rewire('../../es/indexer');
+const { Search } = require('../../es/search');
+const models = require('../../models');
+const { fakeGranuleFactoryV2, fakeCollectionFactory } = require('../../lib/testUtils');
+const { bootstrapElasticSearch } = require('../../lambdas/bootstrap');
+const Stats = require('../../es/stats');
+
+const collectionTable = randomId('collectionsTable');
+const granuleTable = randomId('granulesTable');
+
+process.env.system_bucket = randomId('systemBucket');
```

Comment:
I'm not 100% sure but I think this gets you in trouble with localstack.  try `systembucket`

Reply:
Probably doesn't matter in this case but 
```
>  awslocal s3 mb s3://LocalStack
make_bucket: LocalStack
>  awslocal s3 ls
2020-02-26 15:38:40 localbucket
2020-02-26 21:38:34 localstack
```

Reply:
It looks like it's fine since it lowercases both directions and and can delete the bucket after.